.github/ISSUE_TEMPLATE/dependency_upgrade.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Dependency Upgrade
+about: Upgrade a dependency in the project
+labels: ['kind/dependency', 'status/triage']
+---
+
+### Which dependency needs to be upgraded?
+
+### What is the current version of the dependency?
+
+### Which version to upgrade to?
+
+### Changelog link
+
+Put the dependency changelog link, if there is one available
+
+### :warning: Is this a breaking change?
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Yes
+- [ ] No
+- [ ] Don't know
```

Comment:
This doesn't really make sense for the website since there are no other consumers besides us.